components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/PublisherCommonUtils.java

```diff
@@ -114,7 +114,8 @@
      * @throws APIManagementException If an error occurs while updating the API
      * @throws FaultGatewaysException If an error occurs while updating manage of an existing API
      */
-    public static API updateApi(API originalAPI, APIDTO apiDtoToUpdate, APIProvider apiProvider, String[] tokenScopes)
+    public static API updateApi(API originalAPI, APIDTO apiDtoToUpdate, APIProvider apiProvider, String[] tokenScopes,
+            String organization)
```

Comment:
We don't need separate organization param. We can use organization from originalAPI obj

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10621/commits/d711021b49bdfd26cece08490dee2dcd69b5d626